library/src/webapp/js/portal/portal.sidebar.collapse.js

```diff
@@ -29,11 +29,15 @@ class SidebarCollapseButton {
   }
 
   async setCollapsed(collapsed) {
+    if (!portal.user || !portal.user.id) {
```

Comment:
You should be able to use !portal?.user?.id here.

Reply:
Good one!